### DISCUSSION TITLE
Custom week issue

### DISCUSSION THREAD

**Discussion by ghost** [KadeArchive/Kade-Engine#708] [Q&A]
![Capture](https://user-images.githubusercontent.com/82577051/120890922-531fe700-c638-11eb-927e-dbb999dbe65a.PNG)

im addin a custom week but it says source/StoryMenuState.hx:48: characters 9-10 : Expected ].

If anyone knows the problem, please let me know :) (im new to this crap)

**Comment by puyoxyz** [comment]
show ur code?

**Comment by ghost** [comment]
![Capture2](https://user-images.githubusercontent.com/82577051/120892204-b82b0b00-c63f-11eb-8fec-70b354de1ed4.PNG)

**Reply by puyoxyz** [reply]
you are missing a `,` after senpai array (see how all the first ones have a `,` after)
